realm/realm-library/src/main/cpp/io_realm_RealmFileUserStore.cpp

```diff
@@ -82,7 +84,7 @@ JNIEXPORT jobjectArray JNICALL
 Java_io_realm_RealmFileUserStore_nativeGetAllUsers (JNIEnv *env, jclass)
 {
     TR_ENTER()
-    std::vector<std::shared_ptr<SyncUser>> all_users = SyncManager::shared().all_users();
+    std::vector<std::shared_ptr<SyncUser>> all_users = SyncManager::shared().all_logged_in_users();
```

Comment:
I don't think we need the loop below checking if the state of the user is `SyncUser::State::Active` . [all_logged_in_users](https://github.com/realm/realm-object-store/blob/38e080ff284d17e5fb8955d3dc76aaa60b972acb/src/sync/sync_manager.cpp#L299) already return only `SyncUser::State::Active` users